applications/OptimizationApplication/custom_python/add_custom_response_utilities_to_python.cpp

```diff
@@ -42,6 +43,11 @@ void  AddCustomResponseUtilitiesToPython(pybind11::module& m)
         .def_static("CalculateSensitivity", &LinearStrainEnergyResponseUtils::CalculateSensitivity)
         ;
 
+    py::class_<MaxOverhangAngleResponseUtils >(m, "MaxOverhangAngleResponseUtils")
+        .def_static("CalculateValue", &MaxOverhangAngleResponseUtils::CalculateValue)
+        .def_static("CalculateSensitivity", &MaxOverhangAngleResponseUtils::CalculateSensitivity)
+        ;
```

Comment:
```suggestion
    m.def_submodule("MaxOverhangAngleResponseUtils")
        .def("CalculateValue", &MaxOverhangAngleResponseUtils::CalculateValue)
        .def("CalculateSensitivity", &MaxOverhangAngleResponseUtils::CalculateSensitivity)
        ;
```

It is my bad that I left the other static only reponses with class expsore. We need to change other static only classes also to sub-modules because, the python documentation does not work well for static only classes which does not have `__init__` defined.